stolon/templates/keeper-statefulset.yaml

```diff
@@ -70,7 +85,7 @@ spec:
             value: {{ .Values.pgSuperuserName  | quote}}
           - name: STKEEPER_PG_SU_PASSWORDFILE
             value: "/etc/secrets/stolon/pg_su_password"
-          - name: STSENTINEL_DEBUG
+          - name: STKEPPER_DEBUG
```

Comment:
Typo: Should be `STKEEPER_DEBUG`

Reply:
Fixed typo and added a typo... Fail ... :D